batchflow/models/metrics/classify.py

```diff
@@ -133,6 +132,20 @@ def __init__(self, targets, predictions, fmt='proba', num_classes=None, axis=Non
     def confusion_matrix(self):
         return self._confusion_matrix.sum(axis=0)
 
+    def infmean(self, arr):
+        """
+        We need to average array while skipping infs when there is at least one finite number along averaging axis.
```

Comment:
Change doc to imperative style, e.g. `Compute the arithmetic mean along the axis, ignoring infs. `

Reply:
Fixed.